packages/rn-tester/NativeComponentExample/js/MyNativeView.js

```diff
@@ -85,6 +85,14 @@ function getTextFor(measureStruct: MeasureStruct): string {
     measureStruct.height,
   )}`;
 }
+const opacityDecrementCounter = 0.2;
+
+function getOpacity(opacity: number): number {
+  if (parseFloat(opacity.toFixed(1)) > 0.0) {
+    return opacity - opacityDecrementCounter;
+  }
+  return 1.0;
+}
```

Comment:
can we use a `useState`, similarly to what I did at line 102?
Also, I don't see where we are setting the new value for opacity.

The opacity starts at 1, when we call setOpacity we invoke getOpacity and we decrement it by 0.2. But the next time the button is pressed again, the oiginal opacity value is still 1.

Reply:
If you check the line 103 these is already a `useState` for opacity.
setOpacity is being used here to set the state opacity. And we are using getOpacity just to return a value for next opacity.

Reply:
@cipolleschi, please look into this. Let me know if you need anything from my side.

Reply:
Sorry, my bad, I misread the code.